browser/extensions/extensibles/content/PrivateTab.jsm

```diff
@@ -400,19 +400,25 @@ const PrivateTab = {
   },
 
   togglePrivate(aWindow, aTab = aWindow.gBrowser.selectedTab) {
+    let newTab;
     let { gBrowser } = aWindow;
     aTab.setAttribute("isToggling", true);
     let shouldSelect = aTab == aWindow.gBrowser.selectedTab;
-    let newTab = gBrowser.duplicateTab(aTab);
-    if (shouldSelect) {
-      let gURLBar = aWindow.gURLBar;
-      let focusUrlbar = gURLBar.focused;
-      gBrowser.selectedTab = newTab;
-      if (focusUrlbar) {
-        gURLBar.focus();
+    try {
+      newTab = gBrowser.duplicateTab(aTab);
+      if (shouldSelect) {
+        let gURLBar = aWindow.gURLBar;
```

Comment:
Do we have a policy on destructuring, as is done on 403/404? Ditto line 411. Perhaps there's a complication I'm missing.
Also, I think it's common practice to prefer `const` over `let` - not sure about this code base though.

Reply:
Line 410 should definitely be `let { gURLBar } = aWindow` at which point it would make sense to simply make line 404 `const { gBrowser, gURLBar } = aWindow`. Based on the Mozilla linter, it also seems that we should no longer be using Hungarian notation - do you think I should have a separate PR to handle changing from Hungarian notation the code that we manage? https://firefox-source-docs.mozilla.org/code-quality/lint/linters/eslint-plugin-mozilla/no-aArgs.html

Reply:
I think a separate PR would be better for merging.